homeassistant/components/automation/blueprints/motion_light.yaml

```diff
@@ -0,0 +1,17 @@
+blueprint:
+  name: Light on when motion sensor activated
+  domain: automation
+  input:
+    motion_entity:
+    light_entity:
+trigger:
+  platform: state
+  entity_id: !placeholder motion_entity
+  from: "off"
+  to: "on"
+action:
+  - service: light.turn_on
+    entity_id: !placeholder light_entity
+  - delay: 120
```

Comment:
Since the default is `single` mode, this will cause warnings in the logs on re-triggering. I guess we should add `max_exceeded` = `silent` to this blueprint?

Reply:
Also, the re-trigger might not work as expected on this one... I guess it needs `mode: restart`? Imho that would make more sense for a motion sensor blueprint. Keep the lights on until the last motion detected was 120 seconds ago.